modules/ROOT/pages/ai-azure.adoc

```diff
@@ -0,0 +1,211 @@
+
+include::partial$misc/admon-ai-proxy.adoc[]
+
+== String response
+
+This example demonstrates how to integrate the {pluginname} plugin with the https://learn.microsoft.com/en-us/azure/ai-services/openai/reference#chat-completions[Azure OpenAI Service Chat completions API] to generate a string response.
+
+[source,js]
+----
+// This example stores the API key in the client side integration. This is not recommended for any purpose.
+// Instead, an alternate method for retrieving the API key should be used.
+const AZURE_OPENAI_API_KEY = '<INSERT_API_KEY_HERE>';
+const AZURE_OPENAI_ENDPOINT = '<INSERT_ENDPOINT_URL_HERE>'; // e.g. https://<INSERT_RESOURCE_NAME_HERE>.openai.azure.com/openai/deployments/<INSERT_DEPLOYMENT_ID_HERE>/chat/completions?api-version=<INSERT_API_VERSION_HERE>
+
+const ai_request = (request, respondWith) => {
+  const azureOpenAiOptions = {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+      'api-key': AZURE_OPENAI_API_KEY
+    },
+    body: JSON.stringify({
+      temperature: 0.7,
+      max_tokens: 800,
+      messages: [{ role: 'user', content: request.prompt }],
+    })
+  };
+  respondWith.string((signal) => window.fetch(AZURE_OPENAI_ENDPOINT, { signal, ...azureOpenAiOptions })
+    .then(async (response) => {
+      if (response) {
+        const data = await response.json();
+        if (data.error) {
+          throw new Error(`${data.error.type}: ${data.error.message}`);
+        } else if (response.ok) {
+          // Extract the response content from the data returned by the API
+          return data?.choices[0]?.message?.content?.trim();
+        }
+      } else {
+        throw new Error('Failed to communicate with the Azure OpenAI API');
+      }
+    })
+  );
+};
+
+tinymce.init({
+  selector: 'textarea',  // change this value according to your HTML
+  plugins: 'ai',
+  toolbar: 'aidialog aishortcuts',
+  ai_request
+});
+----
+
+== Streaming response
+
+This example demonstrates how to integrate the {pluginname} plugin with the https://learn.microsoft.com/en-us/azure/ai-services/openai/reference#chat-completions[Azure OpenAI Service Chat completions API] to generate streaming responses.
+
+[source,js]
+----
+const fetchApi = import("https://unpkg.com/@microsoft/fetch-event-source@2.0.1/lib/esm/index.js").then(module => module.fetchEventSource);
+
+// This example stores the API key in the client side integration. This is not recommended for any purpose.
+// Instead, an alternate method for retrieving the API key should be used.
+const AZURE_OPENAI_API_KEY = '<INSERT_API_KEY_HERE>';
+const AZURE_OPENAI_ENDPOINT = '<INSERT_ENDPOINT_URL_HERE>'; // e.g. https://{your-resource-name}.openai.azure.com/openai/deployments/{deployment-id}/chat/completions?api-version={api-version}
+
+tinymce.init({
```

Comment:
We have two `tinymce.init` here. Should we just use `ai_request` here instead?

Reply:
Oh, I am not sure what happened here. I thought I just copied the code example from one of the original code snippets in the AI plugin docs page and adjusted some of the variables.

I will fix this up 👍